R/error-checks.R

```diff
@@ -287,21 +287,46 @@ logErrorMessage <- function(message, logFolderPath = getwd()) {
 }
 
 validateObservedMetaDataFile <- function(observedMetaDataFile, observedDataFile) {

```

Comment:
@pchelle Where is "validateObservedMetaDataFile" called?

Reply:
It is called when creating SimulationSet objects, so within `SimulationSet$new()`

Reply:
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/06dca9dc36a685a1c42b51e8d1eab92bfd3edc45/R/simulation-set.R#L51